infra-templates/ipsec/2/docker-compose.yml

```diff
@@ -6,16 +6,22 @@ services:
     command: sh -c "echo Refer to router sidekick for logs; mkfifo f; exec cat f"
     network_mode: ipsec
     ports:
-    - 500:500/udp
-    - 4500:4500/udp
+      - 500:500/udp
+      - 4500:4500/udp
+    # Force cni-driver to start first
+    depends_on:
```

Comment:
i'm not sure if we should rely/use `depends_on` as once we launch we don't export this value. 

@alena1108 is there another way to say which services should start first?